### DISCUSSION TITLE
هل هيتم تحديث الاداه لدعم120فريم

### DISCUSSION THREAD

**Discussion by nourvip** [MohamedKVIP/MK-PUBG-Mobile-Tool#6] [General]
هل هيتم تحديث الاداه لدعم120فريم

**Comment by MohamedKVIP** [comment]
ان شاء الله فى اقرب وقت هتكون بتدعم 120 فريم

**Reply by nourvip** [reply]
علي البركه باذن الله

**Comment by nourvip** [comment]
علي البركه باذن الله


‫في الاثنين، 13 مايو 2024 في 11:53 ص تمت كتابة ما يلي بواسطة ‪Mohamed
Kamal‬‏ ***@***.***‬‏>:‬

**Comment by ghost** [comment]
@MohamedKVIP  شكرا كتير على هل برنامج الرائع عنجد مفيد جدا الله يقويك وبأنتظار التحديث لدعم 120 فريم .

**Comment by MohamedKVIP** [comment]
تم التحديث 
يمكنك التحميل من [هنا](https://github.com/MohamedKVIP/MK-PUBG-Mobile-Tool/releases/tag/v1.0.8) او من داخل الاصدار القديم

**Comment by nourvip** [comment]
الاداه هل نزلها تحديث يدعم ال120fps؟

في الأربعاء، ١٥ مايو، ٢٠٢٤ ١١:٤٦ م Mohamed Kamal ***@***.***>
كتب:

**Comment by nourvip** [comment]
لوسمحت ياهندسه مافيش طريقه او برنامج يكون anti drop fps في محاكي جيم لوب
لببجي لاني والله زهقت من الفريمات للي بتسقط دي والتقيطع فيها

في الأربعاء، ١٥ مايو، ٢٠٢٤ ١١:٤٦ م Mohamed Kamal ***@***.***>
كتب:

**Comment by nourvip** [comment]
مافيش حاجه تمنع الدروب فريم في اللعبه يكونanti drop fps

**Reply by code-fanatic** [reply]
Try these settings

# Engine Settings
-⭕ Use DirectX+
- ✔️ Rendering cache
- ❌ force global rendering cache
- ❔ Prioritize discrete graphic card (Enable is you have installed Graphic Card in your PC)
- ❌ Rendering optimizations
- ❌ Enable vertical sync (should not be enabled in gameloop even with high specs)


 Antialiasing : Off
 Memory : (Half of your RAM)
 Processor : (Half of cores of processor)
 Resolution : 1280x720
 DPI: 240

#Game
- SD 720
- Balanced
- 60 FPS

**Comment by nourvip** [comment]
Thanks for the tip!

في الاثنين، ٢٠ مايو، ٢٠٢٤ ٩:٤٦ ص Shuja Ahmed ***@***.***> كتب:
